dev/com.ibm.ws.jdbc_fat_postgresql/test-applications/postgresqlApp/src/jdbc/fat/postgresql/web/PostgreSQLTestServlet.java

```diff
@@ -265,6 +261,7 @@ public void testDefaultAutoCommit() throws Exception {
     // Ensure defaultAutoCommit=false behaves properly across global transaction boundaries.
     // Insert/read data with two different DataSources, expect writes to auto-commit
     @Test
+    @SkipIfSysProp(SkipIfSysProp.OS_IBMI) //Skip on IBM i due to Db2 native driver in JDK
```

Comment:
Similar comment here, add FIXME so we know where to go back and re-enable tests that should run.